staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -1328,15 +1328,52 @@ func newErrWatcher(err error) *errWatcher {
 }
 
 func (c *Cacher) ShouldDelegateExactRV(resourceVersion string, recursive bool) (delegator.Result, error) {
-	return delegator.CacheWithoutSnapshots{}.ShouldDelegateExactRV(resourceVersion, recursive)
+	// Recursive not supported unitl exact RV is implemented for WaitUntilFreshAndGet.
```

Comment:
nit: should be "Not recursive not supported ....", no?

Reply:
Done